buildSrc/src/main/groovy/com.flipkart.varadhi.java-library-conventions.gradle

```diff
@@ -4,4 +4,29 @@ plugins {
 
     // Apply the java-library plugin for API and implementation separation.
     id 'java-library'
+
+    // Apply the maven-publish to allow deploying to maven artifactories
+    id 'maven-publish'
+}
+
+
+publishing {
```

Comment:
This will add publishing task to each subproject ? Not all libraries needs to be published ?

Reply:
any module using java-library-conventions will get published
shall we can create a separate convention extending this for publishable-libraray-convention?

Reply:
Yes, either separate publishing convention or add it specifically to the required modules. May be former is better.

Reply:
created separate publish config